### DISCUSSION TITLE
Plugins Catalog: Customize the Header of Plugins Catalog Pages

### DISCUSSION THREAD

**Discussion by josiahg** [grafana/grafana#48079] [Plugins]
Currently, the header of the details page for a given plugin in the catalog contains a preset set of text. It may contain links to documentation, licenses, etc.

Certain teams (specifically, hosted Grafana) would like to customize this text in some cases.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
